src/StoryAvatar/StoryAvatar.js

```diff
@@ -2,17 +2,18 @@ import "./StoryAvatar.css";
 import React from "react";
 import { Link } from "react-router-dom";
 
-export function StoryAvatar({ user }) {
+export function StoryAvatar({ user, isUserNameNeeded }) {
   const userId = user.user_id;
 
   return (
-    <div className="story-avatar">
+    <div id={isUserNameNeeded? "" : "user-avatar-small"} className="story-avatar">
```

Comment:
Why id not class? This will cause issues because all the small stories will have the same id.